unlock-app/src/components/creator/lock/LockIconBar.js

```diff
@@ -38,7 +41,21 @@ export function LockIconBar({
       <IconBarContainer>
         <IconBar>
           <Buttons.Withdraw as="button" lock={lock} />
-          <Buttons.Edit as="button" />
+          {editing && (
```

Comment:
Is this really the best way to toggle between two buttons?

Reply:
I think this works.